readthedocs/projects/models.py

```diff
@@ -1466,6 +1466,17 @@ class Domain(models.Model):
         help_text=_('Number of times this domain has been hit'),
     )
 
+    # This is used in readthedocsext.
+    ssl_status = models.CharField(
+        _('SSL certificate status'),
+        max_length=30,
+        choices=constants.SSL_STATUS_CHOICES,
+        default=constants.SSL_STATUS_UNKNOWN,
+        # Remove after deploy
+        null=True,
+        blank=True,
```

Comment:
If we are adding a `default=` value, I think that `null=` and `blank=` are not needed. The migration will populate old domains with the default value.

Reply:
We are setting it as null to avoid downtime. Using default isn't done at the db level, so it will cause problems.